src/index.css

```diff
@@ -53,7 +50,7 @@ html {
 }
 
 body {
-	background-color: var(--color-bg);
+	background-color: #0c105f;
```

Comment:
I am only curious, why did you replace the CSS variable to the color code?

Reply:
Because I was tired and had been working for a few hours on this! 🤣 Great catch, I've updated it.